packages/frontend/src/components/campaigns/campaign/rewards/index.tsx

```diff
@@ -19,6 +20,12 @@ interface RewardsProps {
 }
 
 export function Rewards({ from, to, rewards, chainId }: RewardsProps) {
+    const t = useTranslations("allCampaigns.rewards");
+
+    const totalUsdValue = rewards.reduce((sum, reward) => {
```

Comment:
This is not ok. If even one of the rewards has a null `usdValue`, the `totalUsdValue` must be set to null to avoid showing wrong data. Also, you can entirely avoid the `reduce` by accessing the precomputed `rewards.usdValue` property.